Project.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+using namespace std;
+
+class User{
+    public:
+    string Username;
```

Comment:
Encapsulate class member variables and provide getter/setter methods to ensure better data encapsulation and validation. [important]